config/wrf/config.docker.full.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
What is the difference to the docker config file we already have?

Reply:
This uses the full 10km domain. I'm using this for testing, but eventually we probably need to be able to specify/override the domain via the CLI (along with the start/end dates)